atc/db/migration/migrations/1550666656_add_needs_kick_to_jobs.up.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
```

Comment:
Need a corresponding `down` migration for this

Reply:
Good catch 👍